content/home/education.yml

```
@@ -6,31 +6,39 @@ education:
 - title: Tutorials
   content: Access topic-based trainings for free.
   url: https://training.galaxyproject.org
+  icon: fa-solid fa-chalkboard-teacher
```

```suggestion
  icon: fa-solid fa-graduation-cap
```

We use the graduation cap in the masthead already so probably best to stay consistent, despite my earlier comment in chat.